### DISCUSSION TITLE
Databricks Data Intelligence Day Manila

### DISCUSSION THREAD

**Discussion by joshua-lagasca** [DAO-Community/Events#11] [Show and tell]
https://events.databricks.com/data-intelligence-day-manila/

Live from Manila

Discover the Data Intelligence Platform
 
Thursday, April 18, 2024 | 8:15 AM-4:30 PM

Shangri-La The Fort, Manila

Join us for an exploration into the new data intelligence platform that’s fundamentally changing every industry. Now you can democratize access to data companywide, automate manual administration and build AI applications in a turnkey solution.

Through deep-dive sessions with Databricks technical experts and a revealing case study with top brands. you’ll learn how:

 

Your entire organization can gain insights from data using natural language
You can develop generative AI applications on your data — without sacrificing data privacy or control
Data intelligence enhances governance, data engineering, data streaming, data warehousing and user experiences
Featured Speakers

Dr. David Hardoon
Chief Executive Officer

Dheeraj Kumar
Advisor

Proletaryo Cabales
Head of Data Engineering

Cecily Ng
VP & General Manager, ASEAN

Raela Wang
Director of Field Engineering

Max Thöne
Manager, Field Engineering

Puneet Goyal
Manager, Field Engineering

Deepak Sekar
Manager, Field Engineering

Yong Hong Goh
Senior Solutions Architect

Nuwan Ganganath M. A.
Specialist Solutions Architect

Agenda
* Subject to changes

8:15 AM - 8:50 AM	Networking Coffee & Registration
8:50 AM - 9:25 AM	Welcome + Keynote
9:25 AM - 9:45 AM	Customer Sharing By Maya
9:45 AM - 10:10 AM	Simplify Data and AI Governance at Scale With Unity Catalog
10:10 AM - 10:35 AM	Build and Deploy Generative AI Apps With Databricks
10:35 AM - 11:00 AM	Networking Break
11:00 AM - 11:25 AM	AI-Powered Data Warehousing With Databricks SQL
11:25 AM - 11:55 AM	Panel Discussion: Winning the Gen AI Race
12:10 PM - 12:15 PM	Closing Remarks
12:15 PM - 1:00 PM	Networking Lunch
1:00 PM - 4:30 PM	Hands-On Lab: Gen AI and SQL

Featured Sponsors

Event Sponsors may be updated on an ongoing basis. 

AWSPlatinum
AWS Privacy Policy

BlazeclanGold
Blazeclan Privacy Policy

KoantekGold
Koantek Privacy Policy

SearceGold
Searce Privacy Policy

**Comment by joshua-lagasca** [comment]
# Keynote1 - Databricks - The difference is Data Intelligence
Speaker: Reila

Struggle of companies w/ AI

[ **Mosaic AI**<br> ] [ **Delta Live Tables** ] [ **Workflows** ] [ **Databricks SQL** ]
| **Data intelligence Engine** |
| **Unity Catalog** |
| **Delta Lake UniForm** |
| **Open Data Lake** |

# Mosaic AI
## Gen AI
- Custom models
- Model serving
- RAG

## End-to-end AI
- MLOps
- AutoML
- Monitoring

## Short Demo/Sample w/ Databricks Notebook
Prompting to generated code
- accept/reject
- explain

# DBRX
Databricks open source LLM

**Highest prod quality**
Accuracy & Control

**Full model ownership**
you own the model weights & data

**Best price/performance**
2x as smart, 2x more efficient

## Short Demo/Sample Databricks Playground

---

# Keynote2 - Maya - On the Lakehouse journey
Speaker: Pro Cabales

## Lessons Learned
### *01* Solve the **big problems**, but **start small**
- Make things as simple as possible
- ?
- Build on top of your team's strength

### *02* Work in **parts** & **begin early**
- Design data pipelines to work w/ incremental data changes
- The best data transformation is NO TRANSFORMATION!
- ?

### *03* Do **more** with **less**
- ?
- Be cose efficient.
- Democratise access to data to avoid technical debt.

### *04* Keep what you **need**, **let go** of the rest
- Optimise, optimise, optimize!
- Set Data Retention Standards.
- Build a Data Observability strategy.

# **End State**: Simple, Centralized, Collaborative

---

# Keynote3 - Databricks - Unified Data and AI Governance
Speaker: Max Thöne

Today, ^^^ is complex

Organizations need a **unified approach** to ^^^

Databricks Data Intelligence Platform
## Unity Catalog (backbone)
- ?
- ...

## UC unlocks value

? -> Accelerate Innovation
? -> Reduce ...
? -> ?
? -> ?

### Sample - Deploying enterprise-grade governance in a regulated industry [AMGEN]

## UC Features
### Access Control for data and AI
- ?
- ?
- ?

### Centralized search & discovery
- ?
- ?
- ?

### Data Sharing & Collaboration
- ?
- ?
- ?
- ?

## Databricks Marketplace

Providers > Databricks Marketplace < Consumers

### AI models: From Sharing to marketplace

## UC Features cont'd
### Lineage
### AI-powered monitoring & observability

## Databricks Assistant
## AI generated table descriptions

### Demo: Unity Catalog
#### Catalog Explorer

- AI Doc
- AI Comments
- prompt/visualize

---

# Keynote5 - Databricks - Build & Deploy Generative AI Apps w/ Databricks
Speaker: Puneet Goyal

GenAI is changing the future of work
GenAI in production can be difficult & expensive

## Build better GenAI solutions on Databricks
### Complete control
Complete ownership, control, and securirty of the model and the data
### Production quality
Accurate and goverened GenAI apps
### Lower cost

## Building your generative AI solution

- Prompt engineering
- Retrieval augmented generation (RAG)
- Fine-tuning
- Pre-training

## Gen AI
"Your data. Your model."

### Mosaic AI Model Serving
- Custom Models
- Foundation Models APIs
- Proprietary Models

### Vector Search

Serverless vector database that ensures LLMs respond with accurate and timely responses

### Fine-tuning

serverless & auto-provisioned, no need to reserve or pick GPUs
...

### Mosaic AI Pre-training
Up to 7x faster and cheaper training LLMs

## Designing a RAG Application
### Why RAG?
RAG custom-tailors models to YOUR data
### Benefits
- up-to-date custom knowledge
- reduced risk of hallucination
- domain=specoific contextualization
- efficiency & cost-effectiveness
### RAG apps are real-time, batch, and streaming

### RAG combines GenAI apps with your data
User Query > [ Data > GenAI > Result ] > User

### Demo - RAG
(LAB)

1 Data preparation > [ Databricks Embedding Endpoint <> Databricks Vector Search Index ] <> 2 Inferences

## Explore Mosaic AI

---

# Keynote6 - ?
Speaker: ?

...

## We're infusing AI in our engine
- Indexless indices
- Automatic data layout optimization
- Intelligent workload management

## Databricks SQL built for Performant BI

### Databricks SQL serverless
Instant, scalable compute for all DW/BI workloads
### Built from the ground up for best performance
Lightning-fast analytics for all queries
### ...
### Lakehouse Federation
Simplify access and query across systems
### Databricks SQL
First-class SQL experience for everything
like SQL management studio
- SQL query editor
- rich visualizations
- new dashboards

---

Firetalk - 
[\\]: # (EOF)
